.gitignore

```diff
@@ -1,4 +1,4 @@
-.DS_STORE
```

Comment:
You need to change/fix the settings on your local git configuration and/or text editor so that it doesn't change all the line endings in every file. 

Your current settings make it so that the git history and diffs are unreadable, and it will drive fellow developers crazy.